packages/experimental-utils/src/ts-eslint/Rule.ts

```diff
@@ -185,7 +185,7 @@ interface RuleContext<
   /**
    * An object containing parser-provided services for rules
    */
-  parserServices?: ParserServices;
+  parserServices: ParserServices;
```

Comment:
question:
considering these are more general types for eslint, should we keep this as optional?
that way you still have to explicitly handle the case that someone uses a different parser.